src/multisig.rs

```diff
@@ -151,52 +168,58 @@ impl RedeemScriptLayout {
             );
             public_keys
         };
-        // Return parsed script
-        Ok(RedeemScriptLayout {
+        // Returns parsed script.
+        Ok(RedeemScriptContent {
             quorum,
             public_keys,
         })
     }
 }
 
+/// The redeem script builder.
 #[derive(Debug)]
-pub struct RedeemScriptBuilder(RedeemScriptLayout);
+pub struct RedeemScriptBuilder(RedeemScriptContent);
 
 impl RedeemScriptBuilder {
+    /// Creates builder for the given quorum value.
     pub fn with_quorum(quorum: usize) -> RedeemScriptBuilder {
-        RedeemScriptBuilder(RedeemScriptLayout {
+        RedeemScriptBuilder(RedeemScriptContent {
             quorum,
             public_keys: Vec::default(),
         })
     }
 
+    /// Creates builder for the given bitcoin public keys.
     pub fn with_public_keys<I: IntoIterator<Item = PublicKey>>(
         public_keys: I,
     ) -> RedeemScriptBuilder {
         let public_keys = public_keys.into_iter().collect::<Vec<_>>();
         let quorum = public_keys.len();
 
-        RedeemScriptBuilder(RedeemScriptLayout {
+        RedeemScriptBuilder(RedeemScriptContent {
             public_keys,
             quorum,
         })
     }
 
+    /// Adds a new bitcoin public key.
     pub fn public_key<K: Into<PublicKey>>(&mut self, pub_key: K) -> &mut RedeemScriptBuilder {
         self.0.public_keys.push(pub_key.into());
         self
     }
 
+    /// Sets the quorum value.
```

Comment:
Maybe "number of signatures required to spend the input" instead of "quorum value"? It's more verbose.